src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -251,11 +263,24 @@ export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props)
 
   return (
     <>
-      <Card>
-        <CardHeader>
-          <CardHeaderMain>
+      <Card isExpanded={props.simple || selected === NO_TARGET ? undefined : isExpanded}>
+        <CardHeader
+          onExpand={props.simple || selected === NO_TARGET ? undefined : onExpand}
+          isToggleRightAligned={props.simple || selected === NO_TARGET ? undefined : true}
+          toggleButtonProps={
+            props.simple || selected === NO_TARGET
+              ? undefined
+              : {
+                  id: 'toggle-button1',
```

Comment:
Maybe a more specific id and aria-label for this `targetSelect`? Just in case another component uses the same.

Reply:
Might also need to change `'aria-labelledby': 'expandable-card-title toggle-button1',` with the new id too?

Reply:
Oh right, I had that before and then accidentally dropped the change, then partially re-added it... d'oh. Thanks.